src/web/browser/sentryLoader/init.ts

```diff
@@ -0,0 +1,75 @@
+import '../webpackPublicPath';
+import { startup } from '@root/src/web/browser/startup';
+
+import { isAdBlockInUse } from './detectAdBlocker';
+
+const init = async (): Promise<void> => {
+    // Sentry lets you configure sampleRate to reduce the volume of events sent
+    // but this filter only happens _after_ the library is loaded. The Guardian
+    // measures page views in the billions so we only want to log 1% of errors that
+    // happen but if we used sampleRate to do this we'd be needlessly downloading
+    // Sentry 99% of the time. So instead we just do some basic math here
+    // and use that to prevent the Sentry script from ever loading.
+    const randomNumber = Math.floor(Math.random() * 100) + 1; // A number between 1 - 100
+    if (randomNumber === 1) return; // 99% of the time we do nothing here
```

Comment:
I think you’re doing this the wrong way around.

```suggestion
    if (randomNumber !== 1) return; // 99% of the time we do nothing here
```

Reply:
I find quite clear, too:

```
if (Math.random() < 0.99) return; // 99% of the time we do nothing here.
```

Reply:
Really great spot! Thanks!